### PR TITLE
CLOUD-2136 upgrade to 7.1.1.CR2

### PR DESCRIPTION
**Pull request by bdecoste**
Thanks for submitting your Pull Request!

Please make sure your PR meets the following requirements:

- [ ] Pull Request contains description of the issue
- [ ] Pull Request does not include fixes for other issues
- [ ] Attached commits represent units of work and are properly formatted
- [ ] You have read and agreed to the Developer Certificate of Origin (DCO) (see `CONTRIBUTING.md`)
- [ ] Every commit contains `Signed-off-by: Your Name <yourname@example.com>` - use `git commit -s`

### CODE REVIEW

File: modules/datagrid/module.yaml [hunk 1]
```diff
@@ -3,4 +3,3 @@ schema_version: 1
 name: datagrid
 execute:
     - script: install.sh
-    - script: install_patches.sh
```
Comment (slaskawi) [inline modules/datagrid/module.yaml:6]: @bdecoste Could you please remove this files as well?

**Review by slaskawi — CHANGES_REQUESTED** [review]

File: image.yaml [hunk 2]
```diff
@@ -2,15 +2,15 @@ schema_version: 1
 
 name: "jboss-datagrid-7/datagrid71"
 description: "Red Hat JBoss Data Grid 7.1 container image"
-version: "7.1.0"
+version: "7.1.1.ER1"
```
Comment (goldmann) [inline image.yaml]: This should be `7.1.1`.

File: image.yaml [hunk 3]
```diff
@@ -2,15 +2,15 @@ schema_version: 1
...
 from: "jboss/openjdk18-rhel7:1.0"
 labels:
     - name: "org.jboss.product"
       value: "datagrid"
     - name: "org.jboss.product.version"
-      value: "7.1.0.GA"
+      value: "7.1.1.ER1"
```
Comment (goldmann) [inline image.yaml]: This should be `7.1.1.GA`.

File: image.yaml [hunk 4]
```diff
@@ -40,12 +40,9 @@ modules:
...
           - name: datagrid
 artifacts:
-    - path: jboss-datagrid-7.1.0-server.zip
-      description: "Artifact is available on Customer Portal: https://access.redhat.com/jbossnetwork/restricted/softwareDetail.html?softwareId=50851&product=data.grid&version=&downloadType=distributions"
-      md5: a814ca5a3c55b450b1a68c13e28ffece
-    - path: jboss-datagrid-7.1.0-CVE-2017-12629-server-patch.zip
-      description: "Artifact is available on Customer Portal: https://access.redhat.com/jbossnetwork/restricted/softwareDetail.html?softwareId=50851&product=data.grid&version=&downloadType=distributions"
-      md5: 1373127f88ab53e1669a37c6817f7217
+    - path: jboss-datagrid-7.1.1.ER1-server.zip
+      description: "http://download-node-02.eng.bos.redhat.com/devel/candidates/JDG/JDG-7.1.1.ER1/jboss-datagrid-7.1.1.ER1-server.zip"
```
Comment (goldmann) [inline image.yaml]: This should not be exposed/added. This is an internal link.

**Review by goldmann — CHANGES_REQUESTED** [review]

**Comment by slaskawi** [comment]
Should be closed in favor of https://github.com/jboss-container-images/jboss-datagrid-7-image/pull/24. See https://issues.jboss.org/browse/CLOUD-2136.

**Comment by goldmann** [comment]
Will be done as part #24.

**Pull request closed without merging**